cmd/tether/ops_linux.go

```diff
@@ -118,16 +118,8 @@ func (t *operations) Setup(sink tether.Config) error {
 		return err
 	}
 
-	// symlink /etc/mtab to /proc/mounts
-	var err error
-	if err = tether.Sys.Syscall.Symlink("/proc/mounts", "/etc/mtab"); err != nil {
```

Comment:
It's moved into the base image instead of created for every container so it doesn't show up in diffs.